src/pool-bin/BinMigrator.sol

```diff
@@ -55,7 +55,8 @@ contract BinMigrator is IBinMigrator, BaseMigrator, ReentrancyLock {
             deltaIds: v4PoolParams.deltaIds,
             distributionX: v4PoolParams.distributionX,
             distributionY: v4PoolParams.distributionY,
-            to: v4PoolParams.to
+            to: v4PoolParams.to,
+            hookData: new bytes(0)
```

Comment:
follow the same as CLMigrator where hookData is `new bytes(0)`

Reply:
Should we support passing-in hookData here ? I mean for both pool types

Reply:
Maybe in separate PR because it's out of scope